modules/nf-core/mmseqs/tsv2exprofiledb/main.nf

```diff
@@ -0,0 +1,45 @@
+process MMSEQS_TSV2EXPROFILEDB {
+    tag "$db"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::mmseqs2=14.7e284" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mmseqs2:14.7e284--pl5321hf1761c0_0':
+        'quay.io/biocontainers/mmseqs2:14.7e284--pl5321hf1761c0_0' }"
+
+    input:
+    path db
+    val db_name
```

Comment:
Can't this be automatically detected by the script?

Reply:
Fixed here [3ca60c1](https://github.com/nf-core/modules/pull/2600/commits/3ca60c1fc7f47cee419a4485e4eed3c48db81efc)

Reply:
Awesome!!